File: rankcUp/forLoop/step1.rb
# 正整数 n と、 n 個の整数 a_1, ..., a_n が半角スペース区切りで与えられます。
# 与えられた整数の中に 3 の倍数がいくつあるかを数え、出力してください。

# 入力される値
# 入力は以下のフォーマットで与えられます。

# n
# a_1 a_2 ... a_n

# 1 行目に正整数 n が、 2 行目に n 個の整数 a_1, ..., a_n が半角スペース区切りで与えられます。

# 期待する出力
# 与えられた整数の中に3の倍数がいくつあるかを出力してください。
# 末尾に改行を入れ、余計な文字、空行を含んではいけません。

# 条件
# すべてのテストケースにおいて、以下の条件をみたします。

# ・n および各 a_i (1 ≤ i ≤ n） は 1 以上 50 以下の整数

n = gets.chomp.to_i
array = gets.chomp.split(" ").map(&:to_i)
odd_number = array.count{ |x| x % 3 == 0 }
puts odd_number

# 回答2
n = gets.to_i
array = gets.chomp.split(" ", n).map(&:to_i)

ans = 0

for num in array
  if num % 3 == 0
    ans += 1
  end
end

puts ans

File: Python/ArrayUsage/OperationsOnAllElements/step1.py
# 配列 A の要素数 N と整数 K, 配列 A の各要素 A_1, A_2, ..., A_N が与えられるので、配列 A に K がいくつ含まれるか数えてください。



n, k = map(int,input().split())

count = 0
for _ in range(n):
    num = int(input())
    if num == k:
        count += 1
print(count)


# 解答2
n, k = map(int, input().split())
a = [int(input()) for _ in range(n)]
ans = 0

for i in a:
    if i == k:
        ans += 1

print(ans)